packages/code-du-travail-frontend/pages/dossiers/[slug].js

```diff
@@ -106,16 +101,39 @@ DossierThematique.getInitialProps = async ({ query: { slug } }) => {
   return { dossier };
 };
 
-const { breakpoints, fonts, spacings } = theme;
+const Category = ({ id, icon, title, shortTitle, refs = [] }) => {
+  const [areAllVisible, setAllVisible] = useState(false);
+  const visibleRefs = refs.slice(0, areAllVisible ? refs.length : 3);
+  return (
+    <StyledWrapper>
+      {icon ? (
+        <IconStripe centered icon={icons[icon]}>
+          <Title as="h3" stripe="none" id={id} data-short-title={shortTitle}>
+            {title}
+          </Title>
+        </IconStripe>
+      ) : (
+        <Title as="h3" stripe="none" id={id} data-short-title={shortTitle}>
+          {title}
+        </Title>
+      )}
+      <StyledFlatList>
+        {visibleRefs.map((ref) => (
+          <Li key={ref.url || ref.externalUrl}>
+            <DossierLink {...ref} />
+          </Li>
+        ))}
+      </StyledFlatList>
+      {!areAllVisible && refs.length > 4 && (
+        <SeeMore small variant="flat" onClick={() => setAllVisible(true)}>
```

Comment:
I guess the <ViemMore /> component does not fit the case.

Reply:
Do you prefer that I call it `SeeAll` ? It's only a `Button` so not sure I understand

Reply:
Sorry I forget to quote `jsx`
I was refering to the `<ViewMore />` component  which handle "view more" list

Reply:
I completly forgot that one ! Maybe that I should add it to the UI lib ?

Reply:
yes why not :)